ksql-engine/src/main/java/io/confluent/ksql/function/udf/geo/GeoDistanceKudf.java

```diff
@@ -44,20 +50,28 @@
   private static final List<String> VALID_RADIUS_NAMES_KMS =
       Lists.newArrayList("km", "kilometer", "kilometers", "kilometre", "kilometres");
 
+  @Udf(description = "The 2 input points should be specified as (lat, lon) pairs, measured"
+      + " in decimal degrees. An optional fifth parameter allows to specify either \"MI\" (miles)"
+      + " or \"KM\" (kilometers) as the desired unit for the output measurement. Default is KM.")
+  public Object geoDistance(
+      @UdfParameter(description = "The latitude of the first point in decimal degrees.")
+        final double lat1,
+      @UdfParameter(description = "The longitude of the second point in decimal degrees.")
+        final double lon1,
+      @UdfParameter(description = "The latitude of the first point in decimal degrees.")
+        final double lat2,
+      @UdfParameter(description = "The longitude of the second point in decimal degrees.")
+        final double lon2,
+      @UdfParameter(description = "The chosen Earth radius.")
+        final String... radius) {
```

Comment:
instead of using variable-arguments here I recommend polymorphism (i.e. have two methods, one with the radius and one without). This should fully support backwards compatibility